include/Metabox/Posts/Tag.php

```diff
@@ -30,11 +30,11 @@ public function render() {
 			<table class="optiontable">
 				<tr>
 					<td scope="row" >
-						<?php $this->render_tags_dropdown( 'smbm_mt_old_tag', $tags ); ?>
+						<?php $this->render_tags_select( 'smbm_mt_old_tag' );?>
```

Comment:
If you look a little above this function, we have already retrieved tags.

So let's pass tags to this function and treat it as an optional parameter. If it is present then we will use it. Otherwise we can retrieve it.

Refer to the existing `reder_tags_select` function to see how to do it.

Also you may have to change the existing `render_tags_select` function instead of creating a new one.

Reply:
@sudar i am not sure is it correct. 

I have to remove existing `render_tags_select` function on Base.php file. Because its not used anywhere.